wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/DtoConverter.java

```diff
@@ -42,27 +44,26 @@
 
 // TODO! use global registry for DTO converters
```

Comment:
Please remove that line. It is not needed. It won't help us to implement global registry.